collect_app/src/main/res/values/styles.xml

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <style name="TextAppearanceLabelExtraLarge" parent="TextAppearance.Material3.LabelLarge">
```

Comment:
This should probably live in `typography.xml`

Reply:
Fixed.